cmd/podman/images/scp.go

```diff
@@ -61,16 +68,38 @@ func scp(cmd *cobra.Command, args []string) (finalErr error) {
 		// TODO add tag support for images
 		err error
 	)
+	for i, val := range os.Args {
+		if val == "image" {
+			break
+		} else if i == 0 {
+			continue
+		} else if strings.Contains(val, "CIRRUS") {
```

Comment:
What is this about?

Reply:
this is how I got the global options into the new command, in the testing suite it was including flags and variables that were causing all sorts of failures. excluding any flag that has CIRRUS in the name fixed those, one of the biggest one was the commit message that was messing everything up.

Reply:
would be nice to have a code comment here to that effect.